package.json

```diff
@@ -19,7 +19,7 @@
     "gen-robot": "node server/gen-robot-txt.js",
     "gen-sitemap": "node server/gen-sitemap-xml.js",
     "gen-lambda": "node server/cloudfront-gen-lambda.js",
-    "lint": "lerna run lint",
+    "lint": "eslint *.js test templates server packages",
```

Comment:
We can add `--fix` option so that eslint can correct some errors automatically. What do you think of ?